packages/runner-ct/src/app/app.tsx

```diff
@@ -47,20 +47,30 @@ const App: React.FC<AppProps> = observer(
       }
     }, [])
 
+    const runSpec = (spec: Cypress.Cypress['spec']) => {
+      setIsSpecsListOpen(false)
+      state.setSingleSpec(spec)
+    }
+
     return (
       <>
         <main className="app-ct">
           <div className={cs('specs-list-container', { 'specs-list-container__open': isSpecsListOpen })}>
             <nav>
-              <a onClick={() => setIsSpecsListOpen(!isSpecsListOpen)} id="menu-toggle"
-                className="menu-toggle" aria-label="Open the menu">
+              <a
+                onClick={() => setIsSpecsListOpen(!isSpecsListOpen)}
+                id="menu-toggle"
+                className="menu-toggle"
+                role='toggle-menu'
+                aria-label={isSpecsListOpen ? 'Close the menu' : 'Open the menu'}
```

Comment:
thought this was pretty nice

Reply:
I agree accessibility is always a good thing.